docs/usage/installation.md

```diff
@@ -11,14 +11,19 @@ Other installation locations are also possible, but please keep the original lay
 
 ## From Godot's Asset Library
 
-_To be written..._
+_Godot-Python is in early development and is not yet available on the Godot Asset Library._
```

Comment:
Minor thing, but I think I'd prefer the name remain lower case: "godot-python" or "godot-python-extension"